lib/web_ui/lib/src/engine/util.dart

```diff
@@ -12,7 +12,7 @@ typedef Callback<T> = void Function(T result);
 ///
 /// Return value should be null on success, and a string error message on
 /// failure.
-typedef Callbacker<T> = String Function(Callback<T> callback);
+typedef Callbacker<T> = String/*?*/ Function(Callback<T> callback);
```

Comment:
Do you mean to keep comment syntax here?

Reply:
For now. This PR only migrates `dart:ui`. `dart:_engine` will follow next.